torch/_dynamo/symbolic_convert.py

```diff
@@ -540,7 +534,13 @@ def generic_jump(truth_fn: typing.Callable[[object], bool], push: bool):
     def jump_graph_break(self, inst, value, extra_msg=""):
         log_graph_break(self.code_options, reason="Data-dependent jump")
         if not self.should_compile_partial_graph():
-            unimplemented("should_compile_partial_graph=False")
+            unimplemented_v2(
+                gb_type="Should not compile partial graph (data-dependent jump)",
+                context="",
+                explanation="Dynamo has determined when encountering a data-dependent "
```

Comment:
How does this manifest from the user side? Do you have example python code that would demonstrate this? I personally do not know what a "data-dependent jump" is so I'm thinking users won't either.